src/Validator/EventListener/ValidateListener.php

```diff
@@ -35,7 +35,7 @@ final class ValidateListener
     private $validator;
     private $resourceMetadataFactory;
 
-    public function __construct(ValidatorInterface $validator, ResourceMetadataFactoryInterface $resourceMetadataFactory)
+    public function __construct(ConstraintValidatorInterface $validator, ResourceMetadataFactoryInterface $resourceMetadataFactory)
```

Comment:
This is a backward compatibility break no?

Reply:
The new `ConstraintValidatorInterface` is extending the actual `ValidatorInterface`. Therefore it's ok no?

Reply:
No, @soyuka is right, this is a BC.
See for instance:
https://3v4l.org/H9fWU